src/server/NetworkingServer.cpp

```diff
@@ -165,10 +170,30 @@ void NetworkingServer::HandleApiCommand(
 
 void NetworkingServer::DeleteClient(sf::TcpSocket* client, std::list<sf::TcpSocket*> clients) {
```

Comment:
@SourenP we gotta get more consistent. in some place functions_are_snake_case in others they are camelCase and here they're SomethingElse

Reply:
Please fix anything you notice

I'm naming member functions and variables camelCase and inline variables, arguments and parameters in snake_case